modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/TreeFilter/FrontendTreeFilter/TreeFilter.js

```diff
@@ -194,7 +194,7 @@ const TreeFilter = ({
 					fluid
 				>
 					<div className="container p-0">
-						<p className="m-0 text-2">
```

Comment:
Why do you remove this utility class?

Reply:
I was not aware of this change. Maybe I missed it in a conflicting rebase. Thanks for the heads up!